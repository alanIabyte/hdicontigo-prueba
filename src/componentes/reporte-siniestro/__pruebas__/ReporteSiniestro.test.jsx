import React from "react";
import renderer from "react-test-renderer";
import ReporteSiniestro from "../reporte-siniestro-componente/ReporteSiniestro";

describe("#ReporteSiniestro", () => {
  it("se despliega correctamente", () => {
    const tree = renderer.create(<ReporteSiniestro />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
