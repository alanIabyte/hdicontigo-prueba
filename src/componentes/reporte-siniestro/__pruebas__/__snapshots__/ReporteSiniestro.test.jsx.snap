// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#ReporteSiniestro se despliega correctamente 1`] = `
<div
  className="sc-ikkxIA"
>
  <div
    className="sc-kpDqfm sc-dAbbOL bTCkFB hdZOgq"
  >
    Número de reporte
  </div>
  <div
    className="sc-kpDqfm sc-feUZmu bTCkFB jMhLTa"
  >
    
  </div>
  <div
    className="sc-kpDqfm sc-dAbbOL bTCkFB hdZOgq"
  >
    Hora
  </div>
  <div
    className="sc-kpDqfm sc-feUZmu bTCkFB jMhLTa"
  >
    Fecha inválida
  </div>
  <div
    className="sc-kpDqfm sc-dAbbOL bTCkFB hdZOgq"
  >
    Siniestros
  </div>
  <div
    className="sc-kpDqfm sc-feUZmu bTCkFB jMhLTa"
  >
    
  </div>
  <div
    className="sc-kpDqfm sc-dAbbOL bTCkFB hdZOgq"
  >
    Ajustador
  </div>
  <div
    className="sc-kpDqfm sc-feUZmu bTCkFB jMhLTa"
  >
    
  </div>
  <div
    className="sc-kpDqfm sc-dAbbOL bTCkFB hdZOgq"
  >
    Arribo de ajustador
  </div>
  <div
    className="sc-kpDqfm sc-feUZmu bTCkFB jMhLTa"
  >
    Fecha inválida
  </div>
</div>
`;
